src/reducers/index.js

```diff
@@ -58,6 +62,14 @@ function getInitialState() {
       subRowHeight: 0,
       subRowHeightGetter: () => 0,
     },
+    columnSettings: {
+      bufferColCount: undefined,
+      colAttributesGetter: undefined,
+      colWidth: 0,
+      colWidthGetter: () => 0,
```

Comment:
This is unused until https://github.com/karry08/fixed-data-table-2/pull/2/ is ported.